src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const elements = document.querySelectorAll('span.population');
+const total = document.querySelector('span.total-population');
+const avgPop = document.querySelector('span.average-population');
+
+let sum = 0;
+let elementsCount = 0;
+
+for (const element of elements) {
+  sum += +element.textContent.replace(/,/g, '');
+  elementsCount++;
+}
+
+const avg = sum / elementsCount;
```

Comment:
Ensure that 'elementsCount' is not zero before performing the division to avoid division by zero errors. Consider adding a check to handle cases where there are no elements.